docs/auth/bitbucketServer/provider.md

```diff
@@ -39,12 +46,34 @@ The Bitbucket Server provider is a structure with two configuration keys:
 To add the provider to the frontend, add the `bitbucketServerAuthApi` reference and `SignInPage` component as shown
 in [Adding the provider to the sign-in page](../index.md#sign-in-configuration).
 
+## Adding the provider to the Backstage backend
+
+To add the provider to the backend, install both the Auth plugin and Bitbucket Server provider module in your backend instance:
+
+```bash title="from your Backstage root directory"
+yarn --cwd packages/backend add @backstage/backend-defaults @backstage/plugin-auth-backend @backstage/plugin-auth-backend-module-bitbucket-server-provider
+```
+
+After that, configure the plugin and module in your backend `index.ts` file:
+
+```ts title="packages/backend/src/index.ts"
```

Comment:
We can simplify this to just be the new `backend.add()` like we have done here: https://backstage.io/docs/auth/bitbucket/provider#backend-installation